app/src/main/java/org/simple/clinic/monthlyReports/questionnaire/Questionnaire.kt

```diff
@@ -0,0 +1,55 @@
+package org.simple.clinic.monthlyReports.questionnaire
+
+import android.os.Parcelable
+import androidx.room.Dao
+import androidx.room.Entity
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.PrimaryKey
+import androidx.room.Query
+import io.reactivex.Observable
+import kotlinx.parcelize.Parcelize
+import org.simple.clinic.monthlyReports.questionnaire.sync.QuestionnairePayload
+import org.simple.clinic.util.Unicode
+import java.util.UUID
+
+@Entity(tableName = "Questionnaire")
+@Parcelize
+data class Questionnaire(
+    val uuid: UUID,
+
+    @PrimaryKey
+    val questionnaire_type: QuestionnaireType,
+
+    val layout: String,
+) : Parcelable {
+
+  fun toPayload(): QuestionnairePayload {
+    return QuestionnairePayload(
+        uuid = uuid,
+        questionnaireType = questionnaire_type,
+        layout = layout)
+  }
+
+  override fun toString(): String {
+    return "Questionnaire(${Unicode.redacted})"
+  }
```

Comment:
This should not contain any personal health information (PHI). So, we don't need to redact the `toString`. We redact data so that it doesn't show up in our error logs on Sentry.